src/app/components/timeline/LineTitleCell.js

```diff
@@ -219,9 +234,14 @@ class LineTitleCell extends PureComponent {
             <Glyphicon glyph="tint" />
           </Button>
           {isSmall ? null : (
-            <Button block bsSize="small" onClick={this.toggleLine}>
-              {expandedIcon}
-            </Button>
+            <ButtonGroup>
```

Comment:
does this look good as a `ButtonGroup`? I think it would look the same as two buttons next to each other, right?

Reply:
I put it them in the group to keep them within a parent element, but on closer inspection, in vertical it gets that -1px margin that throws it. So I should have just asked what type of parent you recommend.